scripts/envs/audio-to-text

```diff
@@ -3,7 +3,7 @@
 # env var used only for this sample you can list it here
 export APP_NAME="audio-to-text"
 export APP_DISPLAY_NAME="Audio to Text Application"
-export APP_DESC="Audio to Text Application example with AI enabled audio transcription"
+export APP_DESC="Build your own AI-enabled audio transcription application. Pick from the model servers available in our catalog or bring your own."
```

Comment:
My only concerns are:

1. do we want to stick with "catalog" since its not available right now
2. some templates dont have the option of bringing your own model server, users are stuck with whatever comes with the template.

Reply:
otherwise i think the desc sounds nice

Reply:
> do we want to stick with "catalog" since its not available right now

+1 good point. I think better to go with `dropdown list` or `list`. wdyt?

> some templates dont have the option of bringing your own model server, users are stuck with whatever comes with the template.

I think we can cover this inside the scope of this PR as we're updating the descriptions individually for each sample. Do you have in mind a sample that we need to update its description?

I might be missing context here, but I see the [audio-to-text](https://github.com/redhat-ai-dev/ai-lab-template/blob/main/templates/audio-to-text/template.yaml#L47) has the option to bring your own model.

Reply:
cc @MariaLeonova

Reply:
@thepetk @maysunfaisal I would suggest to say: `pick from the model servers available` and not specify further, that should be sufficient, IMO.

Reply:
@MariaLeonova let me know if the suggestion below works for you
```suggestion
export APP_DESC="Build your own AI-enabled audio transcription application. Pick from the model servers available or bring your own."
```

Reply:
Looks perfect, thank you!